src/python/tests/storage/test_storage_api.py

```diff
@@ -0,0 +1,283 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+import pytest
+import os
+import py
+from foglamp.storage.payload_builder import PayloadBuilder
+from foglamp.storage.storage import Storage
+
+__author__ = "Vaibhav Singhal"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+# TODO: remove once FOGL-510 is done
+@pytest.fixture(scope="module", autouse=True)
+def create_init_data(request):
+    """
+    Module level fixture that is called once for the test
+        Before the tests starts, it creates the init data
+        After all the tests, it clears database and sets the init data
+    Fixture called by default (autouse=True)
+    """
+    _dir = os.path.dirname(os.path.realpath(__file__))
+    file_path = py.path.local(_dir).join('/foglamp_test_storage_init.sql')
+    os.system("psql < {} > /dev/null 2>&1".format(file_path))
+    yield
+    os.system("psql < `locate foglamp_ddl.sql | grep 'FogLAMP/src/sql'` > /dev/null 2>&1")
+    os.system("psql < `locate foglamp_init_data.sql | grep 'FogLAMP/src/sql'` > /dev/null 2>&1")
+
+
+@pytest.allure.feature("api")
+@pytest.allure.story("storage")
+class TestStorageRead:
+    """
+    This class tests SELECT (Read) queries of Storage layer using payload builder
+    """
+    def test_select(self):
+        result = Storage().query_tbl_with_payload("statistics", PayloadBuilder().SELECT().payload())
```

Comment:
kindly hold payload in a var; it will help to assert/ debug payload in case of future enhancements, bug fixes! `paylaod = PayloadBuilder().SELECT().payload()`